### ISSUE TITLE
@quasar/app-vite - Trailing comma breaks imports from quasar

### ISSUE THREAD

**Issue by joelwat** [quasarframework/quasar#13123]
### What happened?

A trailing comma in a quasar import statement results in a build error.

### What did you expect to happen?

The app to build and run

### Reproduction URL

https://github.com/joelwat/quasar-import-bug

### How to reproduce?

1. `git clone https://github.com/joelwat/quasar-import-bug`
2. `npm install`
3. `npx quasar dev`

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite)

### Quasar info output

```Shell
Operating System - Linux(5.17.1-1-default) - linux/x64
NodeJs - 16.14.2

Global packages
  NPM - 8.5.0
  yarn - 1.22.18
  @quasar/cli - undefined
  @quasar/icongenie - Not installed
  cordova - Not installed

Important local packages
  quasar - 2.6.5 -- Build high-performance VueJS user interfaces (SPA, PWA, SSR, Mobile and Desktop) in record time
  @quasar/app-vite - 1.0.0-beta.13 -- Quasar Framework App CLI with Vite
  @quasar/extras - 1.13.5 -- Quasar Framework fonts, icons and animations
  eslint-plugin-quasar - Not installed
  vue - 3.2.31 -- The progressive JavaScript framework for building modern web UI.
  vue-router - 4.0.14
  pinia - Not installed
  vuex - Not installed
  vite - 2.9.1 -- Native-ESM powered web dev build tool
  eslint - 8.12.0 -- An AST-based pattern checker for JavaScript.
  electron - Not installed
  electron-packager - Not installed
  electron-builder - Not installed
  register-service-worker - 1.7.2 -- Script for registering service worker, with hooks
  @capacitor/core - Not installed
  @capacitor/cli - Not installed
  @capacitor/android - Not installed
  @capacitor/ios - Not installed

Quasar App Extensions
  *None installed*

Networking
  Host - hal
  br0 - 10.0.0.250
```

### Relevant log output

```Shell
Thu, Apr 7, 2022 10:20:00pm ~/programming/bug-reproduction/import-transform-bug                                                                                                                                                               
 (trunk) via  v16.14.2 $ npx quasar dev

 .d88888b.
d88P" "Y88b
888     888
888     888 888  888  8888b.  .d8888b   8888b.  888d888
888     888 888  888     "88b 88K          "88b 888P"
888 Y8b 888 888  888 .d888888 "Y8888b. .d888888 888
Y88b.Y8b88P Y88b 888 888  888      X88 888  888 888
 "Y888888"   "Y88888 "Y888888  88888P' "Y888888 888
       Y8b


 » Reported at............ 4/7/2022 10:20:09 PM
 » App dir................ /home/joel/programming/bug-reproduction/import-transform-bug
 » App URL................ http://localhost:9000/
                           http://10.0.0.250:9000/
                           http://172.16.0.2:9000/
                           http://10.20.0.250:9000/
 » Dev mode............... spa
 » Pkg quasar............. v2.6.5
 » Pkg @quasar/app-vite... v1.0.0-beta.13
 » Browser target......... es2019|edge88|firefox78|chrome87|safari13.1

 App • Opening default browser at http://localhost:9000/

Unknown import from Quasar: 
10:20:10 PM [vite] Internal server error: Unknown import from Quasar: 
  Plugin: vite:quasar:script
  File: /home/joel/programming/bug-reproduction/import-transform-bug/src/App.vue
      at Object.module.exports [as default] (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/quasar/dist/transforms/import-transformation.js:6:11)
      at /home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:116:86
      at Array.map (<anonymous>)
      at /home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:108:10
      at String.replace (<anonymous>)
      at vueTransform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:105:6)
      at TransformContext.transform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:263:17)
      at Object.transform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/vite/dist/node/chunks/dep-611778e0.js:36533:53)
      at async doTransform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/vite/dist/node/chunks/dep-611778e0.js:55507:29)
10:20:12 PM [vite] Internal server error: Unknown import from Quasar: 
  Plugin: vite:quasar:script
  File: /home/joel/programming/bug-reproduction/import-transform-bug/src/App.vue
      at Object.module.exports [as default] (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/quasar/dist/transforms/import-transformation.js:6:11)
      at /home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:116:86
      at Array.map (<anonymous>)
      at /home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:108:10
      at String.replace (<anonymous>)
      at vueTransform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:105:6)
      at TransformContext.transform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/@quasar/vite-plugin/dist/index.js:263:17)
      at Object.transform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/vite/dist/node/chunks/dep-611778e0.js:36533:53)
      at async doTransform (/home/joel/programming/bug-reproduction/import-transform-bug/node_modules/vite/dist/node/chunks/dep-611778e0.js:55507:29)
```

### Additional context

I believe this is the same bug as in https://github.com/quasarframework/quasar/issues/10049 but was reintroduced when @quasar/app was split. I used the fix from https://github.com/quasarframework/quasar/issues/10049 to fix this one

**Comment by yusufkandemir** [comment]
Reproduced the problem with `@quasar/app-vite` v1.0.0-beta.13. It works fine with the latest version which is v1.4.3, but should be working fine with pretty much any version now.

The problem was fixed in [`@quasar/vite-plugin-v1.0.10`](https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fvite-plugin-v1.0.10).

**Comment by rstoenescu** [comment]
Just upgrade all your Quasar packages. Especially ‘@quasar/app-vite’
